tests/terraform/createcluster/createcluster_test.go

```diff
@@ -26,84 +23,101 @@ func Test_TFClusterCreateValidation(t *testing.T) {
 }
 
 var _ = Describe("Test:", func() {
+
 	Context("Build Cluster:", func() {
 		It("Starts up with no issues", func() {
-			status, err := terraform.BuildCluster(&testing.T{}, *tfVars, false)
+			status, err := BuildCluster(&testing.T{}, false)
 			Expect(err).NotTo(HaveOccurred())
 			Expect(status).To(Equal("cluster created"))
+
 			defer GinkgoRecover()
-			fmt.Println("Server Node IPS:", terraform.MasterIPs)
-			fmt.Println("Agent Node IPS:", terraform.WorkerIPs)
-			terraform.PrintFileContents(terraform.KubeConfigFile)
-			Expect(terraform.MasterIPs).ShouldNot(BeEmpty())
-			if terraform.NumWorkers > 0 {
-				Expect(terraform.WorkerIPs).ShouldNot(BeEmpty())
+
+			fmt.Println("Server Node IPS:", MasterIPs)
+			fmt.Println("Agent Node IPS:", WorkerIPs)
+			fmt.Println(KubeConfigFile)
+
+			Expect(MasterIPs).ShouldNot(BeEmpty())
+			if NumWorkers > 0 {
+				Expect(WorkerIPs).ShouldNot(BeEmpty())
 			} else {
-				Expect(terraform.WorkerIPs).Should(BeEmpty())
+				Expect(WorkerIPs).Should(BeEmpty())
 			}
-			Expect(terraform.KubeConfigFile).ShouldNot(BeEmpty())
+			Expect(KubeConfigFile).ShouldNot(BeEmpty())
 		})
 
 		It("Checks Node and Pod Status", func() {
 			defer func() {
-				_, err := terraform.Nodes(terraform.KubeConfigFile, true)
+				fmt.Printf("\nFetching node status\n")
```

Comment:
Let's move this print out of the `defer` block. The `defer` here is just printing the nodes and pods after it checks (whether it passes or fails). Basically doing a `kubectl get nodes,pods -A -o wide`. I'm fine also if we just want to remove this print entirely.